resources/guides/uwinsite_guide.md

```diff
@@ -0,0 +1,236 @@
+
+In the **Summary By Due Date** section, you'll see a breakdown of any charges you owe and the date they are due. In the **Detail Of Charges Due** section, you'll see the description of the fees you owe, the charge date, the term and the charge amount. 
+
+#### Account Services 
+
+The **Account Services** tab on the left has four subcategories:
++ **Charges Due:** provides a summary by due date of the charges and deposits that you owe
++ **Opt Out Drug and Dental:** gives you the option to opt out of your drug and dental insurance 
++ **Income Tax Forms:** allows you to request an income tax T2202A or T4A form by selected year.
+    ![img](../../static/img/uwinsite_guide/income_tax_forms.jpg)
+    1. Select a year
+    2. Click the dropdown arrow to select an income tax form type
+    3. Click on **download**
++ **Online Refund Request:** allows you to request a refund form from the university
+
+## Academics
+
+### My Academics
+
+Here, you'll see your **Academic Summary**, **Enrollment Deadlines**, **Student Evaluation of Teaching**, **a Change of Program Application**, **a Returning Student Application**, **Service Letters**, and **Apply to Graduate**
+
+#### Student Academic Summary
+
+Your summary is broken down into your academic structure. It'll give you a rundown of what you're studying at the University. This section goes into detail regarding your term and when you're expected to graduate.
+
+Further down, you'll see your **term summary**. You can click on the term you wish to view in detail and it will give you the option to add or change a minor. You can also view the details regarding the number of credits of any courses, you dropped, wait-listed, or that you're currently enrolled in.
+
+#### Enrollment Deadlines
+
+Here, you can view your appointments and enrollment dates for another term including the session, when the appointment starts, and the last date to enroll.
+
+#### Change of Program Application
+
+Here, current students can change their programs and this is also for students who applied for part-time studies who wish to apply for full-time studies. You'll also see the deadlines of the application for each term.
+
+To start a change of program application: 
+1. Click **Create New Application**. You will see a variety of areas that need to be completed. 
+2. Now, click the **Application Start** button which you may need to scroll down to see. 
+![img](../../static/img/uwinsite_guide/change_program_app.jpg)
+3. Then, complete the required fields on the page. 
+4. Once you complete a page of your application, click the **Save** button to save what you've filled 
+5. Click the **Continue** button to proceed to the next page and do the same until all pages have been filled.
+6. When you complete your application, click **Submit Application** 
+
+**Note**: *Make sure you save your application often. You can also save what you've currently filled and come back another time to complete the rest of your application.*
```

Comment:
this would be better as a [docusaurus admonition](https://docusaurus.io/docs/markdown-features/admonitions)

I think this goes for any note in this document